test/bin/mdx-test/expect/simple-mli/test-case.mli

```diff
@@ -0,0 +1,9 @@
+(** This is a doc comment with some code blocks in it:
+
+    {[# List.map (fun x -> x * x) [(1 + 9); 2; 3]
+- : int list = [100; 4; 9]
```

Comment:
Looking at the test I'm wondering if it would hurt to have mdx respect the indentation or at least just indent everything to the block start level so that mdx users that don't use ocamlformat or don't have ocp-indent checked in CI can still reasonably use that feature.

That's still probably a lot of people so I'm kind of considering it. How much work do you think that would require?

Reply:
I've done this because I agree it's better for those users. It wasn't a huge amount of work, just a lot of fiddly changes to get it right.